service/component-samples/http-device-sample/pom.xml

```diff
@@ -193,11 +193,17 @@
         <configuration>
           <filesets>
             <fileset>
-              <directory>${project-demo-directory}/${project.artifactId}</directory>
+              <directory>${project-demo-directory}/${project.artifactId}/lib</directory>
```

Comment:
Please raise a different PR for this fix as this is not related to README.

Reply:
Removed from this PR.